python/triqs/sumk/sumk_discrete_from_lattice.py

```diff
@@ -46,7 +47,7 @@ def __init__(self, lattice, patch = None, n_points = 8, method = "Riemann"):
         self.SL = lattice
         self.patch,self.method = patch,method
         # init the array
-        SumkDiscrete.__init__ (self, dim = self.SL.dim, gf_struct = lattice.OrbitalNames)
+        SumkDiscrete.__init__ (self, dim = self.SL.bl.ndim, gf_struct = lattice.OrbitalNames)
```

Comment:
```suggestion
        SumkDiscrete.__init__ (self, dim = self.SL.ndim, gf_struct = lattice.OrbitalNames)
```

Reply:
Thank you for the fix! This class did indeed go unnoticed in the `TBLattice` changes.
I only noticed the simplification above.

A test would certainly be great to have for the `SumkDiscreteFromLattice`!